.github/workflows/automerge.yaml

```diff
@@ -0,0 +1,63 @@
+# This Workflow automatically merges pull requests, submitted by certain users
+# if they pass certain criteria.
+name: Automerge version bumps
+
+on:
+  pull_request:
+    branches: [ master ]
+    paths:
+    # Trigger the workflow only for certain files. First we need to exclude all
+    # files, then add the files we care about.
+    - "!**"
+    - "cockroachdb/Chart.yaml"
+    - "cockroachdb/README.md"
+    - "cockroachdb/values.yaml"
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  automerge:
+    runs-on: ubuntu-latest
+    if: ${{ github.actor == 'cockroach-teamcity' }}
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          # Fetch all branches so it is possible to checkout the default branch files.
+          fetch-depth: 0
+      # The next steps tries to reproduce the steps taken to generate the
+      # files. We restore the changed files to their original state on the
+      # default branch, run the commands to regenerate the change, and verify
+      # the result matches the PR contents (git diff does not show any changes).
+      - name: Regenerate the patch
```

Comment:
nit: maybe name this "confirm expected patch" or something?

(then under "verify nothing changed" move the build steps in there, i.e.)

>  bazel build //build
> $(bazel info bazel-bin)/build/build_/build bump "$appVersion"

Reply:
I'll rename the step. 👍 
The build steps generate the patch, so they kind of belong to this step IMO.

Reply:
ah, gotcha. cool, works for me :) 👍🏼